README.md

```diff
@@ -22,6 +22,9 @@
     <a href="//www.youtube.com/watch?v=1APxaOC9U-A" target="_blank">
         <img src="https://img.shields.io/badge/video-YouTube-red">
     </a>
+    <a href="https://join.slack.com/t/ask-prior/shared_invite/zt-oq4z9u4i-QR3kgpeeTAymEDkNpZmCcg" target="_blank">
+        <img src="https://img.shields.io/static/v1?label=questions&message=Ask PRIOR Slack&color=<COLOR>">
+    </a>
```

Comment:
This looks like it is failing to render:
https://github.com/allenai/ai2thor-rearrangement/tree/adding-leaderboard

![image](https://user-images.githubusercontent.com/28768645/114092817-2dac7200-986f-11eb-8b3f-eb7af0bcc1b2.png)